packages/models-library/tests/test_generics.py

```diff
@@ -39,20 +46,87 @@ def test_dict_base_model():
     assert some_instance["a new key"] == 23
 
 
-def test_data_enveloped(faker: Faker):
-    some_enveloped_string = Envelope[str]()
-    assert some_enveloped_string
-    assert not some_enveloped_string.data
-    assert not some_enveloped_string.error
-
-    random_float = faker.pyfloat()
-    some_enveloped_float = Envelope[float](data=random_float)
-    assert some_enveloped_float
-    assert some_enveloped_float.data == random_float
-    assert not some_enveloped_float.error
-
+def test_enveloped_error_str(faker: Faker):
     random_text = faker.text()
     some_enveloped_bool = Envelope[bool](error=random_text)
     assert some_enveloped_bool
     assert not some_enveloped_bool.data
     assert some_enveloped_bool.error == random_text
+
+
+@pytest.fixture
+def builtin_value(faker: Faker, builtin_type: type) -> Any:
+    return {
+        "str": faker.pystr(),
```

Comment:
wonder if hypothesis could be used for such tests.

Reply:
probably! I will give it a try

Reply:
Too convoluted for this simple case ... it's not worth this time.